scrabble.js

```diff
@@ -1,16 +1,163 @@
+const SCORE = {
+  a: 1,
+  b: 3,
+  c: 3,
+  d: 2,
+  e: 1,
+  f: 4,
+  g: 2,
+  h: 4,
+  i: 1,
+  j: 8,
+  k: 5,
+  l: 1,
+  m: 3,
+  n: 1,
+  o: 1,
+  p: 3,
+  q: 10,
+  r: 1,
+  s: 1,
+  t: 1,
+  u: 1,
+  v: 4,
+  w: 4,
+  x: 8,
+  y: 4,
+  z: 10
+}
+
 
 const Scrabble = {
+
   score(word) {
+    if ( word === "" ) throw "empty";
+    if ( word.length > 7 ) throw "too many letters";
+    if ( word.match(/[^a-zA-Z]/) ) throw "invalid characters";
+
+    let total = 0
+
+    if (word.length === 7) {
+      total += 50
+    }
 
+    for (let i = 0; i < word.length; i += 1) {
+      let letter = word.charAt(i).toLowerCase()
+      total += SCORE[letter]
+    }
+    return total
```

Comment:
You did well with semi-colons at the beginning of this function but lost it by the end!